### PR TITLE
Remove extra fields from playlist items

### PR DESCRIPTION
**Pull request by alex-deaconu**
Remove subscription status descriptions: http://screencast.com/t/WoK4at2rF.

[stage-2]

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks.

File: js/services/svc-presentation-parser.js [hunk 1]
```diff
@@ -16,11 +16,23 @@ angular.module('risevision.editorApp.services')
     'recurrenceMonthOfYear', 'visibility',
     'transition', 'items'
   ])
+  .constant('PLAYLIST_ITEM_JSON_FIELDS', [
+    'name', 'duration', 'type', 'objectReference',
+    'index', 'playUntilDone', 'objectData',
```
Comment (ezequielc) [inline js/services/svc-presentation-parser.js]: @alex-deaconu You mentioned you were removing the index field. Should we keep it here or remove as well? Thanks

Comment (alex-deaconu) [inline js/services/svc-presentation-parser.js]: @ezequielc good point. Once the fix for https://github.com/Rise-Vision/rva/issues/123 is deployed, we can remove the `index` field from here. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu Ok. :+1:

**Pull request merged**